lib/plugins/aws/deploy/compile/events/apiGateway/lib/validate.test.js

```diff
@@ -401,6 +401,7 @@ describe('#validate()', () => {
       headers: ['Content-Type', 'X-Amz-Date', 'Authorization', 'X-Api-Key', 'X-Amz-Security-Token'],
       methods: ['OPTIONS', 'POST'],
       origins: ['*'],
+      allowCredentials: false,
```

Comment:
Could we do the same like proposed above (for the other test) and remove this configuration so that we test for the default case (which should set `allowCredentials` to `false`).